client/tests/integration_test.rs

```diff
@@ -8,10 +8,8 @@ use tower::builder::ServiceBuilder;
 use warp::{Filter, Rejection};
 use warp_json_rpc::Builder;
 
-use casper_node::crypto::Error::*;
-use hex::FromHexError::*;
-use std::time::Duration;
-use Error::*;
+use casper_node::crypto::Error as CryptoError;
+use hex::FromHexError;
```

Comment:
This should be with the other 3rd party imports.

Reply:
OK, I have to ask about it – why are we introducing our own import format rules that we have to apply manually? What's the point really? When it comes to formatting, the rules should be that it's uniform and automatic. I really don't are how they're sorted, I never look at them.

Reply:
It's been a rule we've applied (reasonably strictly) since before I started here.  It makes sense when you have large groups of imports IMO.

Reply:
I know it's been a rule but I have never understood it.

>  It makes sense when you have large groups of imports IMO.

Why does it make sense? Do you ever read the imports?

Reply:
I don't tend to read the full list of imports, but I sometimes look through them for a specific crate's.  I find that simpler if they're categorised like this.

I also don't overly care about this in terms of the rule itself - I'd be fine with all imports grouped together.  I do care about consistently applying the current style guide though :)